src/bika/lims/browser/publish/emailview.py

```diff
@@ -300,7 +299,14 @@ def email_body(self):
         body = self.request.get("body", None)
         if body is not None:
             return body
-        return self.context.translate(_(self.email_template(self)))
+        setup = api.get_setup()
```

Comment:
I would rely on `api.get_senaite_setup()` instead (the field in the actual setup is a proxy that will eventually removed)

Reply:
I would like to keep the `get_setup` until we moved all setup parameters to the new DX setup. Then we just have to change the API call to return the new DX setup object.